src/python/TaskWorker/Actions/RucioActions.py

```diff
@@ -93,7 +93,7 @@ def createOrReuseRucioRule(self, did=None, grouping=None, activity=None,
             self.logger.debug("List of existing rules for this DID")
             ruleId = None
             for rule in ruleIdGen:
-                self.logger.debug("id: %s account: %s activity %s")
+                self.logger.debug("did: %s account: %s activity: %s", did, account, activity)
```

Comment:
thanks for having spotted the missing values ! (how could it run before ? mistery !)
But this is really an `id` i.e. rule-id not a `did` which represents a data-identified in Rucio (file, dataset, container)